### PR TITLE
Adding support for bigquery

### PR DESCRIPTION
**Pull request by mateuszklimek**

### CODE REVIEW

File: redata/backends/sql_alchemy.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        return from_time
+
+    def to_naive_timestamp(self, from_time):
+        return from_time
+
+    def get_time_to_compare(self, time_interval):
+        before = self.transorm_by_interval(time_interval)
+        return before
+
+    def transorm_by_interval(self, time_interval):
```
Comment (jwarlander) [inline redata/backends/sql_alchemy.py:36]:
```suggestion
    def transform_by_interval(self, time_interval):
```

File: redata/backends/bigquery.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    
+    @staticmethod
+    def datetime_types():
+        return [
+            'TIMESTAMP',
+            'DATETIME'
+        ]
+
+    def get_time_to_compare(self, time_interval):
+        to_compare = self.transorm_by_interval(time_interval)
```
Comment (jwarlander) [inline redata/backends/bigquery.py:36]:
```suggestion
        to_compare = self.transform_by_interval(time_interval)
```

File: redata/backends/sql_alchemy.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        return result
+
+    def get_timestamp(self, from_time):
+        return from_time
+
+    def to_naive_timestamp(self, from_time):
+        return from_time
+
+    def get_time_to_compare(self, time_interval):
+        before = self.transorm_by_interval(time_interval)
```
Comment (jwarlander) [inline redata/backends/sql_alchemy.py:33]:
```suggestion
        before = self.transform_by_interval(time_interval)
```

File: redata/backends/exasol.py [hunk 4]
```diff
@@ -103,26 +103,27 @@ def check_count_per_value(self, table, checked_column, time_interval):
...
             FROM {table.table_name}
-            WHERE [{table.time_column}] {where_timecol}
+            WHERE [{table.time_column}] > now() - {interval_part}
             """,
             fetch_dict=True,
         ).fetchone()
         return SimpleNamespace(**result)
     
-    def check_data_volume_diff(self, table, where_timecol):
+    def check_data_volume_diff(self, table, from_time):
```
Comment (jwarlander) [inline redata/backends/exasol.py:119]:
```suggestion
    def check_data_volume_diff(self, table, from_time):
        interval_part = self.db.make_interval(time_interval)
```

File: redata/backends/exasol.py [hunk 5]
```diff
@@ -103,26 +103,27 @@ def check_count_per_value(self, table, checked_column, time_interval):
...
         result = self.db.execute(
             f"""
             SELECT
                 CAST({table.time_column} AS DATE),
                 count(*) as "count"
             FROM {table.table_name}
-            WHERE [{table.time_column}] {where_timecol}
+            WHERE [{table.time_column}] >= '{from_time}'
```
Comment (jwarlander) [inline redata/backends/exasol.py:126]:
```suggestion
            WHERE [{table.time_column}] >= '{interval_part}'
```

**Review by jwarlander — COMMENTED** [review]
Good stuff!

Just suggested a few minor edits :)

File: redata/backends/exasol.py (hunk 4 above)
Comment (mateuszklimek) [reply redata/backends/exasol.py:119]: Hey, for those 2 suggestions with time_interval, I think in this function `check_data_volume_diff` there is no time_intervals, so I would keep the current version. 
I think the one having time_interval is `check_data_volume`, this one I kept untouched :).

**Pull request merged**